src/main/kotlin/no/nav/helsearbeidsgiver/bro/sykepenger/db/ForespoerselDao.kt

```diff
@@ -69,14 +69,6 @@ class ForespoerselDao(private val db: Database) {
                     activeTransaction = this,
                 )
 
-            if (oppdaterteForespoersler.isEmpty()) {
-                val msg =
-                    "Fant ingen aktive eller besvarte forespørsler for vedtaksperioden $vedtaksperiodeId. " +
-                        "Dette skal kun skje for vedtaksperioder som ikke støttes enda (potensielle) eller som stammer fra før pilot."
-                logger.warn(msg)
-                sikkerLogger.warn(msg)
-            }
-
```

Comment:
Dette logges langt oftere enn vi forutså, og for helt hverdagslige tilfeller. Fjerner loggen for å unngå unyttig støy i warning-loggene.